johnsonandjohnson/Bodiless-JS
# POC NX : Start replacing commands with nx commands

## Changes
* add nx
* example build with caching: https://cloud.nx.app/runs/SMGDXXs8tN

## Test Instructions
Ensure all setup, build, start work as intended.
New command: 'npm run nxbuild' to use caching

## Issues to Resolved
* The Tailwind types package couldn't be found (tailwindcss/tailwind-config).   
  * Switched to tailwindcss types -> https://github.com/tailwindlabs/tailwindcss/pull/7891

## Issues to Resolve still
* Something in postcss causes memory issue and crashes with out of memory.   For hack, I copied out the build:css in bodiless-ui and checked in a pre-built bodiless.index.css
* Memory issues when you start..  Sometimes it works and sometimes it doesn't.
